File: VB/B137591/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace B137591
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			InitializeData()
		End Sub
		Private Sub InitializeData()
			For i As Integer = 0 To 9
				dataTable1.Rows.Add(Nothing, Nothing, Nothing, Nothing)
			Next i
		End Sub
	End Class
End Namespace

File: CS/B137591/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace B137591 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            InitializeData();
        }
        private void InitializeData() {
            for (int i = 0; i < 10; i++)
                dataTable1.Rows.Add(null, null, null, null);
        }
    }
}